src/NuGet.Core/NuGet.Packaging/Signing/Utility/VerificationUtility.cs

```diff
@@ -144,7 +144,7 @@ internal static SignatureVerificationStatusFlags ValidateTimestamp(Timestamp tim
 
                 if (!spec.AllowedHashAlgorithmOids.Contains(signerInfo.DigestAlgorithm.Value))
                 {
-                    issues.Add(SignatureLog.Issue(treatIssuesAsErrors, NuGetLogCode.NU3024, string.Format(CultureInfo.CurrentCulture, Strings.VerifyError_TimestampUnsupportedSignatureAlgorithm, signature.FriendlyName)));
+                    issues.Add(SignatureLog.Issue(treatIssuesAsErrors, NuGetLogCode.NU3024, string.Format(CultureInfo.CurrentCulture, Strings.VerifyError_TimestampUnsupportedDigestAlgorithm, signature.FriendlyName)));
```

Comment:
You are passing the signature.FriendlyName here but there's no placeholder for it in the message.

Reply:
Thanks! I'll fix it.